.circleci/config.yml

```diff
@@ -26,13 +26,48 @@ jobs:
           name: code-coverage
           command: bash <(curl -s https://codecov.io/bash)
 
+      - run:
+          name: commitlint
+          command: |
+            echo "module.exports = {extends: ['@commitlint/config-conventional']}" > commitlint.config.js
+            npm install @commitlint/cli @commitlint/config-conventional
+            npx semantic-commitlint -- --ci
```

Comment:
What's the action that a developer can take if this command fails?
How exactly does this help?

Reply:
They rebase to update the commit.   https://help.github.com/en/articles/changing-a-commit-message